tests/integration/__init__.py

```diff
@@ -166,6 +166,11 @@ def get_unused_localhost_port():
     usock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
     usock.bind(('127.0.0.1', 0))
     port = usock.getsockname()[1]
+    if port in (54505, 54506, 64505, 64506, 64510, 64511):
+        # These ports are hardcoded in the test configuration
+        usock.close()
+        return get_unused_localhost_port()
```

Comment:
Ideally, we wouldn't hard code any ports on the testing configuration, but for now, I'll keep the current hard coded ones.
I will however, get a new port before closing the offending port.

Good?